### PR TITLE
Implement feature described in issue #1

### PR DESCRIPTION
**Pull request by devGTP**
This PR implements the feature described in issue #1. The code was generated using devGPT, a GitHub bot based on GPT-4.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Project Title
+
+Rogue game in Python 3 using tcod library
+
+## Description
+
+This project is about creating a rogue-like game in Python 3 using tcod library and some others. 
+
+## Features
```
Comment (petvas) [inline readme.md:9]: these feature not implemented yet add it to the todo list

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Project Title
+
+Rogue game in Python 3 using tcod library
+
+## Description
+
+This project is about creating a rogue-like game in Python 3 using tcod library and some others.
```
Comment (petvas) [inline readme.md]: can you make this a little bit longer?

**Review by petvas — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (petvas) [inline readme.md]: we have no features yet, I want you to leave features empty and move the following items to the todo list.

**Review by petvas — CHANGES_REQUESTED** [review]

**Comment by devGTP** [comment]
Changes have been made based on the review comments. Please review the updated code.

**Comment by devGTP** [comment]
Changes have been made based on the review comments. Please review the updated code.

**Comment by devGTP** [comment]
Changes have been made based on the review comments. Please review the updated code.

**Pull request closed without merging**